docs/development/system-tests.md

```diff
@@ -76,14 +76,38 @@ of that class, and look at the tests in that module for examples.
 
 This test module contains several JUnit test suites:
 - QuickSystemTestSuite (the default)
-- NightlyFunctionalSystemTestSuite
-- NightlyPerformanceSystemTestSuite
+- SlowSuite1-3
+- ExpensiveSuite1-3
```

Comment:
It might be worth adjusting these suite names to clarify that these are system test suites as well? They'd be SlowSystemTestSuite1-N and ExpensiveSystemTestSuite1-N.